CMakeLists.txt

```diff
@@ -379,3 +380,30 @@ install(
   FILES ${HEADERS}
   DESTINATION "include/libmongoc-${MONGOC_API_VERSION}"
 )
+
+# Define pkg-config files
+set(VERSION "${MONGOC_VERSION}")
+set(prefix "${CMAKE_INSTALL_PREFIX}")
+set(libdir "\${prefix}/lib")
```

Comment:
I think it is there so that CMake doesn't expand it, and instead creates a variable containing a literal `${prefix}`, as this is what is expected in the interpolated .pc file.